.ci.yaml

```diff
@@ -1266,41 +1266,61 @@ targets:
   - name: Linux_android complex_layout_android__compile
     recipe: devicelab/devicelab_drone
     presubmit: false
+    bringup: true
     timeout: 60
     properties:
       tags: >
         ["devicelab"]
       task_name: complex_layout_android__compile
+      dependencies: >-
+        [
+          {"dependency": "open_jdk", "version": "11"}
```

Comment:
You'll also need to override the cache.

```yaml
caches: >-
  [
          {"name":"openjdk","path":"java11"},
  ]
```

Reply:
does it make sense to cover this in the docs too ? https://github.com/CaseyHillers/cocoon/blob/1f1fa81e2588904908d613111a9d72e784cc6e24/CI_YAML.md#upgrading-dependencies

Reply:
Yes, i'll send a PR